src/sas/qtgui/Perspectives/Fitting/UnitTesting/ConstraintWidgetTest.py

```diff
@@ -148,11 +148,11 @@ def testOnTabCellEdit(self, widget, mocker):
 
         # disable the tab
         widget.tblTabList.item(0, 0).setCheckState(0)
-        assert widget.tabs_for_fitting["test_tab"] == False
```

Comment:
This and the 3 other changes in this file should be `_ is False` instead of `not _`. The latter checks the truthiness of the value, while the former matches `False` and only `False`.